src/main/resources/templates/score/team.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="cz" xmlns:th="http://www.thymeleaf.org">
```

Comment:
Tady prosim zmenit cz na cs.

Reply:
opravím - mám to i v dalším html, co jsem dělala předtím, tam to teda spravím taky, ale je to i např. v progress/stage, progress/getHintList  (z těch jsem to oprala :-) ) a možná i v dalších

Reply:
Jo, ono to bude na vice mistech. To se pak opravi postupne, jak se budou predelavat sablony.
Ale v novych PR uz to kontroluju.

Jde o to, ze se jeste snazim delat ty HTML validni podle specifikace a "cz" neni spravny kod pro cestinu a validator to hlasi jako chybu.

Reply:
opraveno